src/widgets/TimerWidget/TimerWidget.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import styles from './TimerWidget.module.scss';
+import { TimerWidgetControls } from 'components';
+
+export const TimerWidget: React.FC = () => {
+  const [time, setTime] = useState(0);
+  const [timerOn, setTimerOn] = useState(false);
+
+  useEffect(() => {
+    let interval: any = null;
```

Comment:
This shouldn't be any. Check the return value of setInterval and use correct type.